code/datums/supply_packs.dm

```diff
@@ -310,17 +310,19 @@ ABSTRACT_TYPE(/datum/supply_packs)
 
 /datum/supply_packs/medicalchems
 	name = "Medical: Medical Reservoir Crate"
-	desc = "x4 Assorted reservoir tanks, x2 Sedative bottles, x2 Hyposprays, 2x Automenders, x1 Syringe Kit"
+	desc = "x4 Assorted reservoir tanks, x2 Sedative bottles, x2 Hyposprays, x1 Auto-menders, x2 Brute/Burn Auto-mender Refill Cartridges, x1 Syringe Kit"
```

Comment:
nitpick 2: should specify that it's 2 each of brute and burn refills, and not 2 brute/burn (i.e.: synthflesh) refills